Content.Shared/Gravity/GravityComponent.cs

```diff
@@ -0,0 +1,10 @@
+using Robust.Shared.GameObjects;
+
+namespace Content.Shared.Gravity
+{
+    [RegisterComponent]
+    public class GravityComponent : Component
```

Comment:
an empty, marker component should be enough?

Reply:
usually marker components are a sign of bad but this seems fine

Reply:
an idea Vera had was having a 'SpaceStationComponent' which you attach to a grid to signify that it's a space station; which would hold stuff like the manifest, station name, and i guess that it has gravity?

Reply:
This also needs to be used for shuttles so having the additional data for them seems silly given it's superfluous.